alacritty_terminal/src/tty/mod.rs

```diff
@@ -70,6 +70,10 @@ pub fn setup_env(config: &Config) {
     // Advertise 24-bit color support.
     env::set_var("COLORTERM", "truecolor");
 
+    // When not on a Windows or macOs system, EGL is preferred over GLX by default. May be
+    // overridden by user's config below.
+    env::set_var("EGL_OVER_GLX", "true");
```

Comment:
Don't set a variable like that, it's user controllable. So completely remove those lines.

Reply:
I noticed that the variable stays set even when removed from the config (and restarting), that is why I added this line. Perhaps it requires a restart of the terminal spawning alacritty to reset properly? I'll remove this anyway if you don't see an issue.

Reply:
Shouldn't be a thing, but switching to a config option easily solves this too.